frontend/src/store/modules/maps.js

```diff
@@ -52,9 +53,15 @@ const getters = {
 };
 
 const actions = {
+
   async getMapsListing({ commit }, model) {
     const payload = { model: model.apiName, version: model.apiVersion };
     const mapsListing = await mapsApi.fetchMapsListing(payload);
+    let avail2D = false;
+    Object.keys(mapsListing).forEach((cat) => {
```

Comment:
Out of curiosity, why `cat`?

Reply:
I was thinking `category`, the `mapListing` has the keys `compartment maps`, `subsystem maps` etc (but now I had to think for some minutes before remembering why I called it `cat`, so maybe it's not a very good name after all... If you have better suggestions, please let me know :))

Reply:
`componentWithMap`?